packages/rspack/src/config/types.ts

```diff
@@ -25,7 +27,7 @@ export type {
 
 export type Configuration = RspackOptions;
 
-export interface RspackOptions {
+export interface LegacyRspackOptions {
```

Comment:
Can you add a deprecation comment so collaborators know why it's called legacy?

Reply:
I removed this type.